diff --git a/node_modules/@polymer/marked-element/marked-import.js b/node_modules/@polymer/marked-element/marked-import.js
index 0842bcf..5915042 100644
--- a/node_modules/@polymer/marked-element/marked-import.js
+++ b/node_modules/@polymer/marked-element/marked-import.js
@@ -8,11 +8,11 @@ found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by Google as
 part of the polymer project is also subject to an additional IP rights grant
 found at http://polymer.github.io/PATENTS.txt
 */
-import * as marked from 'marked/lib/marked.js';
+import { marked } from 'marked';
 
-if (!window.marked) {
+if (!globalThis.marked) {
   // For webpack support for the Polymer 3 version created by the Polymer
   // Modulizer More info:
   // https://github.com/PolymerElements/marked-element/issues/81
-  window.marked = marked;
+  globalThis.marked = marked;
 }
